x-pack/test/performance/journeys/base.config.ts

```diff
@@ -36,7 +36,18 @@ export default async function ({ readConfigFile, log }: FtrConfigProviderContext
     },
     kbnTestServer: {
       ...functionalConfig.get('kbnTestServer'),
-      serverArgs: [...functionalConfig.get('kbnTestServer.serverArgs')],
+      serverArgs: [
+        ...functionalConfig.get('kbnTestServer.serverArgs'),
+        `--telemetry.labels.branch=${process.env.BUILDKITE_BRANCH}`,
+        `--telemetry.labels.ciBuildId=${process.env.BUILDKITE_BUILD_ID}`,
+        `--telemetry.labels.ciBuildJobId=${process.env.BUILDKITE_JOB_ID}`,
+        `--telemetry.labels.ciBuildNumber=${process.env.BUILDKITE_BUILD_NUMBER}`,
+        `--telemetry.labels.git_rev=${process.env.BUILDKITE_COMMIT}`,
```

Comment:
Nit: All other parameters use camel casing. Can we change this to `gitRev` for consistency?

Reply:
Sure, I missed it 👍🏽